### PR TITLE
[DAT-11816] Added foundational compatibility test - manipulating with big queries

### PR DESCRIPTION
**Pull request by AlexanderSashchenko**

### CODE REVIEW

File: src/main/groovy/liquibase/harness/compatibility/foundational/FoundationalCompatibilityTest.groovy [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+        given: "read input data"
+        UIService uiService = Scope.getCurrentScope().getUI()
+
+        Map<String, Object> argsMap = new HashMap()
+        argsMap.put("url", testInput.url)
+        argsMap.put("username", testInput.username)
+        argsMap.put("password", testInput.password)
+        String baseChangelogPath = "liquibase/harness/compatibility/foundational/"
+        String insertChangelog = baseChangelogPath + "insert/" + testInput.change + ".xml"
+        String updateChangelog = testInput.baseChangelogPath.replace("setup", "update")
```
Comment (KushnirykOleh) [inline src/main/groovy/liquibase/harness/compatibility/foundational/FoundationalCompatibilityTest.groovy:42]: this part is frustrating, why we use different approaches for different changetypes?

File: src/main/resources/liquibase/harness/compatibility/foundational/update/firebird/createTable.xml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
+        <sql>UPDATE foundational_test_table SET test_1=50, test_2=49, test_3=48, test_4=47, test_5=46, test_6=45, test_7=44, test_8=43, test_9=42, test_10=41, test_11=40,
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/compatibility/foundational/update/firebird/createTable.xml:8]: plz add comment to this changelog why default one doesn't work

File: src/main/resources/liquibase/harness/compatibility/foundational/update/h2/createTable.xml [hunk 3]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
+        <sql>UPDATE PUBLIC.foundational_test_table SET test_1=50, test_2=49, test_3=48, test_4=47, test_5=46, test_6=45, test_7=44, test_8=43, test_9=42, test_10=41, test_11=40,
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/compatibility/foundational/update/h2/createTable.xml:8]: same, plz add comment to this changelog why default one doesn't work

File: src/main/resources/liquibase/harness/compatibility/foundational/update/derby/createTable.xml [hunk 4]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
+        <loadUpdateData file="update.csv"
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/compatibility/foundational/update/derby/createTable.xml:8]: plz add comment to this changelog why default one doesn't work

File: src/main/resources/liquibase/harness/compatibility/foundational/setup/derby/createTable.xml [hunk 5]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/compatibility/foundational/setup/derby/createTable.xml:7]: plz add comment to this changelog why default one doesn't work

File: src/main/resources/liquibase/harness/compatibility/foundational/setup/h2/createTable.xml [hunk 6]
```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+         http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-latest.xsd">
+    <changeSet id="1" author="as" labels="foundational">
```
Comment (KushnirykOleh) [inline src/main/resources/liquibase/harness/compatibility/foundational/setup/h2/createTable.xml:7]: plz add comment to this changelog why default one doesn't work

**Review by KushnirykOleh — CHANGES_REQUESTED** [review]
overall looks good, just few nits

File: src/main/groovy/liquibase/harness/compatibility/foundational/FoundationalCompatibilityTest.groovy (hunk 1 above)
Comment (AlexanderSashchenko) [reply src/main/groovy/liquibase/harness/compatibility/foundational/FoundationalCompatibilityTest.groovy]: ++

File: src/main/resources/liquibase/harness/compatibility/foundational/update/firebird/createTable.xml (hunk 2 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/compatibility/foundational/update/firebird/createTable.xml:9]: ++

File: src/main/resources/liquibase/harness/compatibility/foundational/update/h2/createTable.xml (hunk 3 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/compatibility/foundational/update/h2/createTable.xml:9]: ++

File: src/main/resources/liquibase/harness/compatibility/foundational/update/derby/createTable.xml (hunk 4 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/compatibility/foundational/update/derby/createTable.xml]: ++

File: src/main/resources/liquibase/harness/compatibility/foundational/setup/derby/createTable.xml (hunk 5 above)
Comment (AlexanderSashchenko) [reply src/main/resources/liquibase/harness/compatibility/foundational/setup/derby/createTable.xml:8]: ++

**Review by KushnirykOleh — APPROVED** [review]
good to merge

**Pull request merged**